pandas/tests/groupby/test_groupby.py

```diff
@@ -2057,20 +2019,21 @@ def get_result(**kwargs):
                 result = get_result()
                 # In this case we have list-of-list, will raise TypeError,
                 # and subsequently be dropped as nuisance columns
-                if op == "sum":
-                    expected = df.set_index(keys)[["C"]]
-                else:
-                    expected = df.set_index(keys)[[]]
```

Comment:
Does this address the FIXME above @jbrockmendel?

Reply:
im not sure what past-me had in mind and can't tell from the GH refs or blame.  i guess the thing to do is manually confirm that the behavior here matches what you think it _should_ be, then remove the comment.

Reply:
Thanks for checking; I think the behavior in this PR looks correct. In all the cases that hit here, the groupby object is operating on the column "C" and this is now coming through to the result.